src/metric/cmake_install.cmake

```diff
@@ -0,0 +1,44 @@
+# Install script for directory: /home/wg25r/new/CMT/src/metric
```

Comment:
This is another file that is auto generated and that we are not saving in the repos.